### PR TITLE
Simplify installation command with bundle add

### PR DESCRIPTION
**Pull request by Spone**
We can now use `bundle add` instead of manually `echo`ing the gem name to `Gemfile`

### CODE REVIEW

**Comment by Spone** [comment]
I signed the CLA but I'm not able to re-run the check.

**Comment by andyw8** [comment]
Thanks @Spone. Let's also update `docs/Quickstart.md` to keep it consistent?

**Comment by andyw8** [comment]
For reference, the `bundle add` command was added in 1.15.0.pre.1 (2017-04-16)

https://github.com/rubygems/bundler/blob/master/CHANGELOG.md

**Comment by Spone** [comment]
@andyw8 done! :)

File: docs/Quickstart.md [hunk 1]
```diff
@@ -43,8 +43,7 @@ $ heroku create name
 Run these commands to add the `shopify_app` Gem to your app:
```
Comment (andyw8) [inline docs/Quickstart.md:43]: I just noticed that this should also be changed, i.e. "Run this command...".

Comment (Spone) [reply docs/Quickstart.md]: Good catch! Fixed.

**Comment by tanema** [comment]
Thanks!

**Pull request merged**